Automattic/vip-go-mu-plugins
# Add filter to allow VIP SMTP Bypass

By default, all VIP environments are automatically configured to send transactional emails (e.g. password reset, new comment) generated by WordPreess using VIP SMTP servers.

For cases, where a third party provider needs to be used for all emails from the site, including both marketing and transaction emails, the new filter (`vip_smtp_enabled`) can be use to disable loading of the VIP SMTP mailer and loading a custom one.

Note that loading a custom one is important otherwise no emails will get sent.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

Need to test this on a sandbox, since local falls back to default mail.

1. Apply patch.
1. Trigger password reset and verify you get email.
1. Add `add_filter( 'vip_smtp_enabled', '__return_false' );`.
1. Trigger password reset and verify you do not get an email.
